### ISSUE TITLE
More runtime compatibility

### ISSUE THREAD

**Issue by Airkro** [sindresorhus/promise-fun#22]
`p-*` function is useful, thank you for your great work.

Recently I had some trouble when I use them in some non `node.js` runtime.

The error cause by:

`p-*` -> https://github.com/sindresorhus/aggregate-error -> https://github.com/sindresorhus/clean-stack/blob/main/index.js#L1

It's possible to stop using `aggregate-error` in `p-*`?

**Comment by sindresorhus** [comment]
The only way I can think of to fix support this is to use an `exports` map in package.json with a separate `browser.js` file for browsers that do not import `os`. Most users use a bundler for browser-based stuff though, so this is not something I plan to work on, but a good pull request would be welcome.

**Comment by sindresorhus** [comment]
> So, it's possible to stop using aggregate-error in p-*?

Yes, when my packages can target Node.js 16 as then there's a native `AggregateError`.
